samples/step-by-step/sample.md

```diff
@@ -7,100 +7,191 @@ redirects:
 - nservicebus/nservicebus-step-by-step-guide
 ---
 
-In this sample shows a very simple ordering system that
+This sample illustrates basic NServiceBus concepts in a step-by-step fashion. Alternatively, the completed sample code can be downloaded above.
 
- * sends a command from a client to a server
- * that server handles the command and publishes a new event about the success
- * a subscriber listens to, and handles, the event
 
+## Concepts
 
-## The Shared Project
+The goal of this step-by-step sample is to show how to build a simple system that uses messaging to:
 
-The `Shared` project is the container for shared classes including message definitions. This project will be shared between the client and server so both sides agree on the typed message descriptions. It is referenced by all projects in the solution.
+ * Send a command from a client to a server
+ * Receive a message from a client on the server that publishes a new event
```

Comment:
this statement seems confusing is it "the server that published the event" or "the server that then publishes a new event"